From 64878063dd63cb0afd8b437554450b84fcc41604 Mon Sep 17 00:00:00 2001
From: Matheus Guilhermino <matheus.machadoguilhermino@windriver.com>
Date: Fri, 27 May 2022 15:38:50 +0000
Subject: [PATCH] Adjust puppetlabs-postgresql version requirement

As can be verified on the module's changelog, support for Debian 11 was
added on puppetlabs-postgresql v7.4.0, which is already out of the
specified range.

Other than added functionality and fixes, here are the major changes
between v6.10.2(latest version inside of range) and v8.0.0:
v7.0.0 drops support for SLES 11 and RHEL 5, and bumps minimum Puppet
version to 6.0.0 (We are currently using Puppet 5.5.22, but it should
be noted that the minimal version was bumped up because Puppet 5 was
removed from the test cases and not because there are signs of
malfunction).
v8.0.0 drops support for CentOS 6, Debian 6, and Ubuntu 10, which is not
a problem since we are not using any of those OSs.

In conclusion, any version earlier than v7.4.0 should not be used and
there are no known disadvantages to using v8.0.0 instead of v7.4.0.

Signed-off-by: Matheus Guilhermino <matheus.machadoguilhermino@windriver.com>
---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index f26057f..6cb103f 100644
--- a/metadata.json
+++ b/metadata.json
@@ -23,7 +23,7 @@
         },
         {
             "name": "puppetlabs/postgresql",
-            "version_requirement": ">=6.4.0 <7.0.0"
+            "version_requirement": ">=6.4.0 <=8.0.0"
         }
     ],
     "description": "Puppet module library to expose common functionality between OpenStack modules.",
-- 
2.30.2

